app/view_objects/articles/social_image.rb

```diff
@@ -34,7 +34,7 @@ def url
     def legacy_article_social_image
       cache_key = "article-social-img-#{article}-#{article.updated_at}-#{article.comments_count}"
```

Comment:
Same as the other PR, can we reformat updated_at here to use `.rfc3339`? Please and THANK YOU!!!!

Reply:
Done :rocket: